### ISSUE TITLE
que improvements

### ISSUE THREAD

**Issue by Mandarjin** [harmonoid/harmonoid#436]
Hi,
I've started using harmonoid recently and this is my only major issue with the program.
I like to shuffle my entire libray at once and then change the order que arround, add or remove songs to the que. However in the current build that I'm using it doesn't seem possible to alter the order of songs in the que, and if you click the add to que button it puts it at the end of the que and not after the currently playing song meaning i will effectively never hear the music i add to the que because of how long it is (over 2.5k at the moment of writing)
TLDR
The ability to change the order of songs in the que by dragging them up and down
Add a setting or change it so that when you use the add to que button it puts the song directly after the currently playing
The ability to remove songs from the que

**Comment by alexmercerind** [comment]
Added.
